source/code-generation/query-converter-test-queries.txt

```diff
@@ -0,0 +1,60 @@
+.. _rm-query-converter-test-bench:
+
+======================
+Test Converted Queries
```

Comment:
I think "Test Queries" would be a better title. It matches the UI, and also encompasses the fact you can test both the original and converted queries.

Reply:
Updated.